source/DmapSupport.cpp

```diff
@@ -327,44 +345,69 @@ HRESULT OpenControllerDevice(PWCHAR deviceName, HANDLE & handle, DWORD shareMode
 #endif // WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
 
 #if !WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)  // If building a UWP app
+
 /**
-Acquire an exclusive access lock on a controller.
+Send an IO control code to the device driver
 \param[in] handle Handle opened to the device to be locked.
```

Comment:
This line should be changed to reflect that this parameter does not relate to a device being locked.